raw_data/preprocessed/africa/sierra_leone_english.txt

```diff
@@ -0,0 +1,132 @@
+
+Environmental standards.
+(34) The Minister may by statutory instrument make regulations establishing national environmental standards for the following: [i] water quality; [ii] effluent limitations; [iii] air quality; [iv] waste; [v] atmospheric protection; [vi] ozone protection; [vii] noise control; [viii] pesticide residues; and [ix] odour. 
+
+Toxic and hazardous substances.
+(35) [1] The Minister may on the advice of the Board prescribe activities or substances which shall be considered hazardous. [2] The Minister shall take all necessary and appropriate measures to monitor, control and regulate the manufacture, sale, transportation, handling or disposal of toxic and hazardous substances, including toxic and hazardous wastes. [3] The introduction or importation of toxic or hazardous wastes into Sierra Leone for storage or disposal by any means whatsoever is prohibited. [4] The possession, introduction or importation into Siena Leone of internationally banned chemicals or substances is prohibited. [5] The discharge of any toxic and hazardous substance into the air or in, on or under the land and waters of Siena Leone is prohibited. [6] Any person who contravenes the provisions of subsection [3], [4] or [5] commits an offence and is liable on conviction to a fine not exceeding two million leones or to a term of imprisonment not exceeding two years or to both the fine and imprisonment. [7] Notwithstanding the provisions of subsection [5], it shall be a defence if it is proved that the discharge was caused solely by a natural disaster or an act of war. [8] Without prejudice to subsection [1], a Court may declare an activity or substance to be hazardous where the Court is satisfied on the evidence before it that the activity or substance has the potential of causing grave damage to the environment.
+
+Notification to the Minister.
+(36) [1] Where a vessel or on shore or offshore facility is used or is involved in the commission of an offence against subsection [5] of the Minister of section 35 the owner or operator of such vessel or onshore or offshore facility shall: [a] promptly notify the Minister or other relevant authority of the discharge; [b] take all possible measures in accordance with standard removal methods to minimize pollution; [c] promptly comply with such other directions as the Minister may from time to time prescribe. [2] An owner or operator who fails to comply with subsection [1] commits an offence and is liable on conviction to a fine not exceeding five million leones or to a term of imprisonment not exceeding two years or to both the fine and imprisonment. 
```

Comment:
Please replace `on shore` with `onshore`. Also, please replace `subsection [5] of the Minister of section 35` with `subsection [5] of section 35`.